core/lib/generators/spree/dummy/templates/rails/test.rb

```diff
@@ -27,7 +27,9 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
-  ActionMailer::Base.default :from => "spree@example.com"
+  ActionMailer::Base.default from: "spree@example.com"
+  config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
```

Comment:
shall we use host: 'test.host' instead of 'localhost:3000', just for better impression